museum/sampleRecords/itemData.destination.json

```diff
@@ -0,0 +1,362 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "object_id":"1",
+      "id":"1",
+      "idno":"A1988.135",
+      "display_label":"STRAWBERRY HILL FARM, ALBANY",
+      "ca_objects.Description":"Harper, Mary Alison. Gilt-painted wood frame, unglazed oil of Strawbwrry Hill Farmhouse, Albany, with grey roof. 3 sections: front: 1 storey: 2x6 paned windows, joining building with door &amp; bell LHS, 2-storey 1x9 paned window &amp; chimney. Post\/rail fence. LHS stump; roof height, gateway,no gate. Signed LR.",
+      "ca_objects.MakersMarks":"Harper, Mary Alison",
+      "ca_objects.ItemDates":"1905-",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Oil on hardboard",
+      "ca_objects.Dimensions.DimensionsValue":"H:58.4cms W:81.8cms",
+      "classificationHierarchy":"ILLUSTRATIONS;Oil;building"
+    },
+    {
+      "object_id":"2",
+      "id":"2",
+      "idno":"A1998.16",
+      "display_label":"GALLOP HOUSE",
+      "ca_objects.Description":"Harper, Mary Alison. Unframed, unglazed oil painting of Gallop House overlooking Swan River. Signed: LRC.  Painted c1974.",
+      "ca_objects.MakersMarks":"Harper, Mary Alison",
+      "ca_objects.ItemDates":"c1974",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Oil on Masonite",
+      "ca_objects.Dimensions.DimensionsValue":"H: 45.4cms W: 55.5cms.  Unframed.",
+      "classificationHierarchy":"ILLUSTRATIONS;Oil;building"
```

Comment:
This `classificationHierarchy` value does not include the root, does this mean "ILLUSTRATIONS" is making a new root?